packages/typespec-python/scripts/eng/regenerate.ts

```diff
@@ -117,9 +121,15 @@ function getEmitterOption(spec: string): Record<string, string>[] {
 }
 
 // Function to execute CLI commands asynchronously
-async function executeCommand(command: string): Promise<void> {
+async function executeCommand(tspCommand: TspCommand): Promise<void> {
+    try {
+        rmSync(tspCommand.outputDir, { recursive: true, force: true });
```

Comment:
We shouldn't introduce `rmSync` into an async function, that def makes time take a lot longer I think